### ISSUE TITLE
Incorrect link to SDK in the repo description

### ISSUE THREAD

**Issue by YohDeadfall** [dotnet/sdk#3451]
Probably it was changed some time ago, since it leads to an example now. The right URL is https://dotnet.microsoft.com/download.

**Comment by livarcocc** [comment]
Where are you seeing the download link above?

**Comment by YohDeadfall** [comment]
In the repo description, in the header:

![Screenshot](https://user-images.githubusercontent.com/10694013/61816865-dde90580-ae55-11e9-84eb-175a49aca8d1.png)

**Comment by livarcocc** [comment]
I don't believe that's necessarily the wrong link though. From there you can get to the download link. I would you expect that to be a download link rather than some more instructional material?

**Comment by YohDeadfall** [comment]
I fixed it in two places in the Core CLR repo, and this one looks wrong too, but maybe was correct some time ago. When an user visit this repo he probably wants to contribute or understand how it works, so a basic example isn't the right thing to point out to an experienced guy. Maybe it makes sense to change the link to the main .NET page (https://dot.net or https://dotnet.microsoft.com).

**Comment by livarcocc** [comment]
There is more detail on the readme below. I find the dot.net links to be far too generic to be a link from this repo. They don't even close on .NET Core, which is what this repo is about.

**Comment by YohDeadfall** [comment]
What's a reason of having any link which doesn't suit well?
